spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java

```diff
@@ -51,6 +51,7 @@
 		ContextDefaultConfigurationRegistrar.class })
 @ConditionalOnClass(name = "com.amazonaws.services.cloudformation.AmazonCloudFormation")
 @EnableConfigurationProperties(AwsStackProperties.class)
+@ConditionalOnProperty(name = "cloud.aws.stack.enabled", havingValue = "true")
```

Comment:
This is one way we can tackle disabling Cloudformation but I am wondering why to even add Cloudformation dependency in first place. 

Wouldn't it make more sense if we mark Cloudformation dependency as optional and enable configuration if user adds it to the classpath? I am thinking about similar approach to other integrations provided in context module.

Reply:
agree on that. I will remove this commit in order to be address the issue later then.